lib/ext/measure.rb

```diff
@@ -14,7 +14,10 @@ def display_name
   end
 
   def cms_int
```

Comment:
This integer's mostly used to sort lists of measures. We shouldn't have measures without CMS IDs, but if we do, they'll just all get grouped together in sorting...